autogen/README.md

```diff
@@ -183,6 +183,24 @@ In order to operate with the Service Account you must activate the following API
 - Compute Engine API - compute.googleapis.com
 - Kubernetes Engine API - container.googleapis.com
 
+## node_pools Variable
+The node_pools variable takes the following parameters:
+
+| Name | Description | Requirement | 
+| --- | --- | --- |
+| name | The name of the node pool. If left blank, Terraform will auto-generate a unique name | Optional |
+| min_count | Minimum number of nodes in the NodePool. Must be >=0 and <= max_count | Required |
```

Comment:
Not required, has a default: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/cluster.tf#L150